### BIG differences, this is diff of wrong files.
### CLUS "Type NIL"
### and CLHS topic selection http://www.lispworks.com/documentation/lw50/CLHS/Body/a_nil.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:nil.txt	2017-04-27 21:39:50.000000000 +0100
+++ saved-pages-clus/cl:types:nil.txt	2017-04-27 21:39:50.000000000 +0100
@@ -1,4 +1,22 @@
-NIL
-Please select which reference to NIL you intended
+Type NIL

-Constant VariableTypeGlossary Entry
\ No newline at end of file
+Supertypes
+all types
+Description
+The type nil contains no objects and so is also called the empty type.
+The type nil is a subtype of every type. No object is of type nil.
+Notes
+The type containing the object nil is the type null, not the type nil.
\ No newline at end of file
